config/admin.config.php

```diff
@@ -1,175 +1,190 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2013 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2013-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
-return array(
-    'router' => array(
-        'routes' => array(
-            'zf-apigility-doctrine-rpc-service' => array(
+namespace ZF\Apigility\Doctrine\Admin;
+
+use ZF\Apigility\Doctrine\Server;
+
+return [
+    'router' => [
+        'routes' => [
+            'zf-apigility-doctrine-rpc-service' => [
                 'type' => 'segment',
```

Comment:
We can't do that. Library is compatible with ZF2 and ZF3, so:
ZF2: `Zend\Mvc\Router\Http\Segment`
ZF3: `Zend\Router\Http\Segment`